File: solutions/typescript/raindrops/1/raindrops.ts
export function convert(n: number): string {
  let text = "";
  if (n % 3 === 0) {
    text += "Pling";
  }
  if (n % 5 === 0) {
    text += "Plang";
  }
  if (n % 7 === 0) {
    text += "Plong";
  }
  if (text.length === 0) {
    text += n;
  }
  return text;
}

File: solutions/scala/raindrops/1/src/main/scala/Raindrops.scala
object Raindrops {
  def convert(n: Int): String = {
    val sb = new StringBuilder()
    if(n % 3 == 0) {
      sb.append("Pling")
    }
    if(n % 5 == 0) {
      sb.append("Plang")
    }
    if(n % 7 == 0) {
      sb.append("Plong")
    }
    if(sb.length == 0) {
      sb.append(n)
    }
    sb.toString
  }
}

